packages/block-library/src/style.scss

```diff
@@ -135,75 +135,75 @@
 	// Our classes uses the same values we set for gradient value attributes, and we can not use spacing because of WP multi site kses rule.
 	/* stylelint-disable function-comma-space-after */
 	.has-vivid-cyan-blue-to-vivid-purple-gradient-background {
-		background: linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%);
+		background: linear-gradient(135deg, rgba(6, 147, 227, 1) 0%, rgb(155, 81, 224) 100%);
```

Comment:
The added spaces look much nicer, but looking at the code comment above, the missing spaces seem to have been added for a reason (kses).